README.md

```diff
@@ -33,7 +33,17 @@ Mount the engine in your `config/routes.rb`:
 mount Moirai::Engine => '/moirai', as: 'moirai'
 ```
 
-and run `bin/rails db:migrate`.
+Generate the necessary files including the database migration:
```

Comment:
Can we also mount the engine automatically? Not for this PR. I'll add it to the questions

Reply:
oooo that would be fancy! let's look in to this